server/conf/i18n/messages

```diff
@@ -564,11 +672,14 @@ validation.impossibleDob=Date of Birth should be less than 150 years ago
 # ENUMERATION QUESTION - text when viewing a question where a user must name entities, such as children. #
 #--------------------------------------------------------------------------------------------------------#
 
+# Text on a button an applicant would click to add another entity
```

Comment:
Here and elsewhere, we don't say what the `{0}` value is. Was that the way the original context string was?